src/app/features/admin-home/store/effects.ts

```diff
@@ -0,0 +1,30 @@
+import { HttpErrorResponse } from "@angular/common/http";
+import { inject } from "@angular/core";
+import { Actions, createEffect, ofType } from "@ngrx/effects";
+import { catchError, map, of, switchMap } from "rxjs";
+import { AdminHomeService } from "../services/admin-home-service";
+import * as AdminHomeActions from "../store/actions";
+
+export const loadVersionInfoEffect = createEffect(
+  (actions$ = inject(Actions), adminHomeService = inject(AdminHomeService)) => {
+    return actions$.pipe(
+      ofType(AdminHomeActions.fetchProbesInfo),
+      switchMap(() => {
+        return adminHomeService.getProbesInfo().pipe(
+          map((data) => {
+            return AdminHomeActions.fetchProbesInfoSuccess({
+              probesInfo: {
+                version: data.version,
```

Comment:
use ?. to check for null/undefined property value

Reply:
done